.github/workflows/docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Documentation
+# This job builds and deploys documenation to github pages.
+# It runs on every push to master.
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    container:
+      image: tendermintdev/docker-website-deployment
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: Install and Build 🔧
+        run: |
+          apk add rsync
+          make build-docs LEDGER_ENABLED=false
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@3.7.1
```

Comment:
What I meant was to update the action with https://github.com/marketplace/actions/netlify-deploy And change the branch name to `netlify-docs`

@marbar3778

Reply:
I understand, but this migrates from github-pages to nelify. Cosmos-sdk and Tendermint are on github-pages, I'm not sure why gaia should be different?

I understand that something may be easier but having a unified system across all repos is better IMO.

Reply:
I agree with @marbar3778 in having a unified system, but also understand @lovincyrus about deploy previews. Fortunately, we can have both worlds, right? We point DNS at github pages and still link Netlify to the repo for deploy previews. Anyway, I think this PR should be merged.

Reply:
Sure, let's do that. For any new changes about Netlify, we can do that another time.